SmartDeviceLink/public/SDLVideoStreamingCapability.m

```diff
@@ -17,7 +17,15 @@
 
 @implementation SDLVideoStreamingCapability
 
+
+// deprecated method, and will be removed in the future
 - (instancetype)initWithPreferredResolution:(nullable SDLImageResolution *)preferredResolution maxBitrate:(int32_t)maxBitrate supportedFormats:(nullable NSArray<SDLVideoStreamingFormat *> *)supportedFormats hapticDataSupported:(BOOL)hapticDataSupported diagonalScreenSize:(float)diagonalScreenSize pixelPerInch:(float)pixelPerInch scale:(float)scale {
+    self = [self initWithPreferredResolution:preferredResolution maxBitrate:maxBitrate supportedFormats:supportedFormats hapticDataSupported:hapticDataSupported diagonalScreenSize:diagonalScreenSize pixelPerInch:pixelPerInch scale:scale preferredFPS:DEFAULT_FRAME_RATE];
+
+    return self;
```

Comment:
When the generated init is used (see comment I left in the header file) you can set `nil` for the `preferredFPS` so you don't need the `DEFAULT_FRAME_RATE` anymore: 

```
self = [self initWithPreferredResolution:preferredResolution maxBitrate:@(maxBitrate) supportedFormats:supportedFormats hapticSpatialDataSupported:@(hapticDataSupported) diagonalScreenSize:@(diagonalScreenSize) pixelPerInch:@(pixelPerInch) scale:@(scale) preferredFPS:nil];
```

Reply:
I have updated SDLVideoStreamingCapability.m based on the generated code as per suggestion.